tests/container

```diff
@@ -231,6 +232,52 @@ else
     waitInstanceBooted c1
 
     lxc exec c1 -- grep -F "delegate_cmds=map_create:prog_load,delegate_maps=ringbuf,delegate_progs=socket_filter,delegate_attachs=cgroup_inet_ingress" /proc/self/mountinfo
+
+	lxc stop -f c1
+fi
+
+echo "==> Test running miscellaneous binary formats with binfmt_misc in unprivileged container"
+if lxc info | grep -F 'unpriv_binfmt: "true"'; then
+	lxc start c1
+	waitInstanceBooted c1
+
+	echo "Enabling binfmt_misc in unprivileged container"
+	lxc exec c1 -- mount binfmt_misc -t binfmt_misc /proc/sys/fs/binfmt_misc
+    [ "$(lxc exec c1 -- cat /proc/sys/fs/binfmt_misc/status)" = "enabled" ]
+
+	echo "Installing required packages"
+    lxc exec c1 -- apt-get update
+    lxc exec c1 -- apt-get install --no-install-recommends -y qemu-user-static binfmt-support
+    lxc exec c1 -- systemctl restart systemd-binfmt # Restart systemd-binfmt service to pick up new binary types provided by qemu-user
+
+	echo "Compiling aarch64 Go test binary on host"
+cat > hello.go << 'EOF'
+package main
+
+import "fmt"
+
+func main() {
+    fmt.Println("Hello, World!")
+}
+EOF
+
+    GOOS=linux GOARCH=arm64 go build -o hello hello.go
+
+    echo "Pushing binary to container"
+    lxc file push hello c1/root/hello
+    lxc exec c1 -- chmod +x /root/hello
```

Comment:
`lxc file push --quiet` here please. Also, `lxc file push` already preserves the execute bit that `go build` have set so no need to `chmod`.